Editor/CoreAPI/StateManager.cs

```diff
@@ -254,6 +254,7 @@ public void SetProfile(string profileName)
             GameLiftWrapper = AmazonGameLiftWrapperFactory.Get(ProfileName);
             FleetManager = new GameLiftFleetManager(GameLiftWrapper);
             ComputeManager = new GameLiftComputeManager(GameLiftWrapper);
+            ResetParameters();
```

Comment:
We should not reset the fleet when switching the profile. Each profile already has a fleet defined for it, we should be correctly displaying that fleet in the UI when we switch profiles

Reply:
Removed